ROADMAP.md

```diff
@@ -1,102 +1,126 @@
 # AWS CDK Roadmap
 
-The [AWS CDK Roadmap] lets developers know about our upcoming features and priorities to help them 
-plan how to best leverage the CDK and identify opportunities to contribute to the project. The roadmap 
-provides a high-level view of our work in progress across the [aws-cdk], [aws-cdk-rfcs], and [jsii] 
-repositories, and creates an opportunity for customers to engage in a conversation with AWS CDK engineers to 
-give us direct feedback.
+The roadmap priorities for the AWS CDK (Cloud Development Kit) are informed by what we hear from customers and interactions on Github, CDK.dev Slack, Stack Overflow, and Twitter. This document outlines the high level direction we are working towards, and for each project there is a tracking issue where you can leave feedback. We update this doc on a quarterly basis to reflect any changing priorities. 
```

Comment:
I think we're missing a sentence introducing what the AWS CDK Roadmap is

Reply:
cdk.dev instead of CDK.dev?

Reply:
also nit: we should split up paragraphs into individual lines so that they are easier to review and read in raw text.